examples/02-inputs/04-forms/src/app.gleam

```diff
@@ -163,24 +175,23 @@ fn view_input(
       attribute.class(
         "block mt-1 w-full px-3 py-1 border rounded-lg focus:shadow",
       ),
-      case state {
-        Ok(_) -> attribute.class("focus:outline focus:outline-purple-600")
-        Error(_) -> attribute.class("outline outline-red-500")
+      attribute.value(form.field_value(form, name)),
```

Comment:
Not sure about that one as it was not there before.

Reply:
Right, this example is demonstrating how to use forms in an _uncontrolled_ manner: we don't want to be setting the `value` here.